core/src/main/java/com/linecorp/armeria/server/PathMapping.java

```diff
@@ -48,14 +50,36 @@
      * Returns the logger name.
      *
      * @return the logger name whose components are separated by a dot (.)
+     *
+     * @deprecated Use {@link RequestLog#name()}, {@link RequestLog#serviceName()} or
+     *             {@link Route#pathPattern()}.
      */
+    @Deprecated
     String loggerName();
 
     /**
      * Returns the value of the {@link Tag} in a {@link Meter} of this {@link PathMapping}.
+     *
+     * @deprecated Use {@link RequestLog#name()}, {@link RequestLog#serviceName()} or
+     *             {@link Route#pathPattern()}.
      */
+    @Deprecated
     String meterTag();
 
+    /**
+     * Returns the path pattern of this {@link PathMapping}. The returned path pattern is different according to
+     * the value of {@link #pathType()}.
+     *
+     * <ul>
+     *   <li>{@linkplain RoutePathType#EXACT EXACT}: {@code "/foo" or "/foo/bar"}</li>
+     *   <li>{@linkplain RoutePathType#PREFIX PREFIX}: {@code "/foo/*"}</li>
+     *   <li>{@linkplain RoutePathType#PARAMETERIZED PARAMETERIZED}: {@code "/foo/:bar" or "/foo/:bar/:qux}</li>
+     *   <li>{@linkplain RoutePathType#REGEX REGEX}: {@code "/*&#42;/foo" }</li>
+     *   <li>{@linkplain RoutePathType#REGEX_WITH_PREFIX REGEX_WITH_PREFIX}: {@code "/foo/*&#42;/bar" }</li>
+     * </ul>
+     */
+    String pathPattern();
```

Comment:
How about renaming this? We currently use the term `pathPattern` for the path patterns a user specify when binding a service and it has different format from this.

Reply:
Users usually access this method by `Route`. Let me think more about which name goes well with `Route`. 🤔

Reply:
How about `patternString()`?

Reply:
That sounds good, I think webflux uses `getPatternString`